go-apps/meep-loc-serv/server/loc-serv.go

```diff
@@ -438,35 +445,41 @@ func checkNotificationRegisteredUsers(oldZoneId string, newZoneId string, oldApI
 
 func sendZonalPresenceNotification(notifyUrl string, notification InlineZonalPresenceNotification) {
 	startTime := time.Now()
-
 	jsonNotif, err := json.Marshal(notification)
 	if err != nil {
 		log.Error(err)
 		return
 	}
+
 	resp, err := http.Post(notifyUrl, "application/json", bytes.NewBuffer(jsonNotif))
+	duration := float64(time.Since(startTime).Microseconds()) / 1000.0
 	_ = httpLog.LogTx(notifyUrl, "POST", string(jsonNotif), resp, startTime)
 	if err != nil {
 		log.Error(err)
+		met.ObserveNotification(sandboxName, serviceName, notifZonalPresence, notifyUrl, nil, duration)
 		return
 	}
+	met.ObserveNotification(sandboxName, serviceName, notifZonalPresence, notifyUrl, resp, duration)
```

Comment:
`met.ObserveNotification(...) call on line 459 & 462 could be a single call after line 455